src/components/FilterGenreOptionsList/FilterGenreOptionsList.js

```diff
@@ -1,19 +1,22 @@
 import React from "react";
-import { useDispatch } from "react-redux";
-import { filterMoviesData } from "../../store/moviesActions";
+import { useNavigate } from "react-router-dom";
 import { movieFilterOptions } from "../../utils/filterOptions";
+
 import "./FilterGenreOptionsList.css";
 
 const FilterGenreOptionsList = () => {
-  const dispatch = useDispatch();
+  const navigate = useNavigate();
   return (
     <ul className="filter-genre">
       {movieFilterOptions.map((filterOption) => (
```

Comment:
please correct movieFilterOptions to make 'all' button work (return to a state when app starts).